src/components/button/_button.scss

```diff
@@ -285,7 +285,6 @@
   }
 
   .#{$prefix}--btn--sm {
-    @include letter-spacing;
```

Comment:
Does this get type styles from `--btn` which is why we're okay removing letter-spacing?

Reply:
Removed `@ letter-spacing` because it's deprecated and there is no successor - I thought that IDL no longer dictates letter-spacing. Do you think otherwise...? Thank you for pointing this out!